server/app/schemas/user.py

```diff
@@ -5,22 +5,25 @@
 from werkzeug.security import generate_password_hash
 
 from app.models import User
+from app.schemas import RoleSchema
 
 
 class UserSchema(Schema):
     """Serialized user schema."""
 
-    id = fields.UUID(dump_only=True)
```

Comment:
Why did you favor String over int or GUID?

Reply:
The id contains letters so it can't be an int and UUID is the only supported field Marshmallow offers which didn't work correctly